### PR TITLE
Fix linter issues using updated dependencies.

### PR DESCRIPTION
**Pull request by josemauro**

### CODE REVIEW

File: backends/influx.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def _query_assemble(clause, namespace, start, end, field=None,
 
 def _validate_namespace(namespace):
     if not isinstance(namespace, str) or not re.match(r'\S+', namespace):
-        error = (f'Error. Namespace should be a string.')
-        raise TypeError
+        error = ('Error. Namespace should be a string.')
+        raise TypeError(error)
 
     if 'kytos.kronos' not in namespace:
         error = (f'Error. Namespace \'{namespace}\' most have the format '
```
Comment (hdiogenes) [inline backends/influx.py:51]: s/most/must/

File: backends/influx.py [hunk 2]
```diff
@@ -44,8 +44,8 @@ def _query_assemble(clause, namespace, start, end, field=None,
 
 def _validate_namespace(namespace):
     if not isinstance(namespace, str) or not re.match(r'\S+', namespace):
-        error = (f'Error. Namespace should be a string.')
-        raise TypeError
+        error = ('Error. Namespace should be a string.')
```
Comment (hdiogenes) [inline backends/influx.py:47]: Parentheses aren't needed.

File: tests/__init__.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 import sys
 from pathlib import Path
 
 BASE_ENV = Path(os.environ.get('VIRTUAL_ENV', '/'))
 
-NAPPS_DIR = BASE_ENV / 'var/lib/kytos/'
+KRONOS_PATH = BASE_ENV / 'var/lib/kytos/napps/..'
 
-sys.path.insert(0, str(NAPPS_DIR))
+sys.path.insert(0, str(KRONOS_PATH))
```
Comment (hdiogenes) [inline tests/__init__.py:10]: This is reverting a change we had previously made to make the code less specific to the kronos NApp.

File: tests/test_main.py [hunk 4]
```diff
@@ -1,14 +1,20 @@
-"""Tests main module of kytos/kronos."""
-from flask import Flask
-from tests.helpers import get_controller_mock
-
-from napps.kytos.kronos.utils import NamespaceError
+"""Module to test main module of kytos/kronos.
 
-# pylint: disable=wrong-import-order,wrong-import-position
+Unit tests to check fail and success cases in main module.
+isort:skip_file
```
Comment (hdiogenes) [inline tests/test_main.py:4]: Can this be a comment instead of a docstring line?

File: setup.py [hunk 5]
```diff
@@ -216,7 +216,7 @@ def symlink_if_different(path, target):
       author_email='of-ng-dev@ncc.unesp.br',
       license='MIT',
       install_requires=[line.strip()
-                        for line in open("requirements/run.in").readlines()
+                        for line in open("requirements/run.txt").readlines()
```
Comment (hdiogenes) [inline setup.py:219]: `run.in` is used to avoid conflicts with kytos core dependencies.

File: tests/test_main.py (hunk 4 above)
Comment (josemauro) [reply tests/test_main.py:4]: No, I've tested and doesn't work and this how the documentation specify it.

File: tests/__init__.py (hunk 3 above)
Comment (josemauro) [reply tests/__init__.py]: Ok, I'll undo it.

File: backends/influx.py (hunk 2 above)
Comment (josemauro) [reply backends/influx.py]: I'll removed it.

File: backends/influx.py (hunk 1 above)
Comment (josemauro) [reply backends/influx.py]: Ok

**Pull request merged**